tests/Doctrine/Tests/Common/Annotations/AbstractReaderTest.php

```diff
@@ -648,3 +648,14 @@ class Name extends Annotation
 {
     public $name;
 }
+
+namespace Doctrine\Tests\Common\Annotations;
+
+use Doctrine\Common\Annotations\Annotation;
+
+if(!class_exists(\Doctrine\Tests\Common\Annotations\Name::class, false)) {
```

Comment:
Wouldn't be simpler to have a file for this class and leave this check for the autoloader (instead of having the class in two different places with this `if`)?